demos/mouse-follow/follow.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"image/color"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+)
+
+type DefaultScene struct{}
+
+type Guy struct {
```

Comment:
Same comment as in
https://github.com/EngoEngine/ecs/pull/18#discussion_r60993404
> Lets make it gender neutral. `Player`, `Hero`, `Character` or something along those lines.